pkg/streaming/commands/processor_list.go

```diff
@@ -120,8 +120,8 @@ func (opts *ProcessorListOptions) print(processor *streamv1alpha1.Processor, _ p
 	row.Cells = append(row.Cells,
 		processor.Name,
 		cli.FormatEmptyString(processor.Spec.FunctionRef),
-		cli.FormatEmptyString(strings.Join(prependParameterNames(processor.Spec.Inputs, processor.Spec.InputNames), ",")),
-		cli.FormatEmptyString(strings.Join(prependParameterNames(processor.Spec.Outputs, processor.Spec.OutputNames), ",")),
+		cli.FormatEmptyString(strings.Join(prependAliases(processor.Spec.Inputs), ",")),
```

Comment:
add a space after the coma to increase legibility, especially now that there is other punctuation in the value

Reply:
👍